spec.html

```diff
@@ -16175,16 +16175,16 @@
               for (let key of Reflect.ownKeys(obj)) {
                 if (typeof key === "string") {
                   let desc = Reflect.getOwnPropertyDescriptor(obj, key);
-                  if (desc && !visited.has(key)) {
+                  if (desc &amp;&amp; !visited.has(key)) {
```

Comment:
Agreed. @shvaikalesh See https://mathiasbynens.be/notes/ambiguous-ampersands for more info on why this is perfectly valid.